spec/features/posts_index_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe 'UserPostsIndex', type: :feature do
+  before do
+    @user = User.create!(name: 'User', email: 'user@example.com')
+
+    5.times { |i| @user.posts.create!(title: "Post #{i + 1}", text: "This is post #{i + 1}") }
+  end
+
+  it 'shows posts of a user' do
+    visit user_posts_path(@user)
+    @user.posts.each do |post|
+      expect(page).to have_content(post.title)
+      expect(page).to have_content(post.text[0..20])
+    end
+  end
+
+  it 'redirects to the post show page on click' do
+    visit user_posts_path(@user)
+    click_on @user.posts.first.title
+    expect(current_path).to eq(post_path(@user.posts.first))
+  end
```

Comment:
- [ ] Here, you only have two tests written whereas, it is expected to be 10. Kindly add the rest 
![image](https://github.com/jlberbesi/blog-app/assets/86200356/6232d652-41b9-40b2-9e39-dd0485110130)

Reply:
![imagen](https://github.com/jlberbesi/blog-app/assets/88105477/9e8c5895-f770-4e80-a4e6-91a698d7c31c)